tools/reviewer/local_server/run.sh

```diff
@@ -1,43 +1,60 @@
 # Run LocalHttpGateway and LocalServer
 # Usage:
-# run.sh --root_path <root_path>
+# run.sh start --root_path <root_path>
```

Comment:
Let's rename `run.sh` to `local_server.sh`, since `run.sh stop` seems confusing.